WalletWasabi.Fluent/Controls/LabelsItemsPresenter.axaml

```diff
@@ -35,14 +34,18 @@
     <Setter Property="Margin" Value="0" />
     <Setter Property="VirtualizationMode" Value="Simple" />
     <Setter Property="HorizontalAlignment" Value="Left" />
+    <Setter Property="BorderBrush" Value="{DynamicResource TagsBoxBorderBrush}" />
+    <Setter Property="Foreground" Value="{DynamicResource TextControlForeground}" />
+    <Setter Property="MaxLabelWidth" Value="120" />
```

Comment:
Did you test all usages of LabelsItemsPresenter, hope it does not break any layout code, this is very sensitive control to such changes.

Reply:
Presumably yes:

- History
![image](https://user-images.githubusercontent.com/3109851/203037680-3ceb277f-5ad0-4f5f-a64e-9b27732e924c.png)
- Label selector
![image](https://user-images.githubusercontent.com/3109851/203037794-98de3629-5b44-4a62-bb9d-660a56378006.png)
- Coin List 
![image](https://user-images.githubusercontent.com/3109851/203037887-414a3d93-1781-47d5-8543-c9a9596e3cb6.png)
- Transaction preview
![image](https://user-images.githubusercontent.com/3109851/203037973-3ef02e85-3521-42ee-971f-f223cb92c36a.png)


Seems to work as expected.